SmartDeviceLinkTests/RPCSpecs/ResponseSpecs/SDLGetVehicleDataResponseSpec.m

```diff
@@ -147,8 +151,10 @@
                                                      SDLRPCParameterNameTirePressure:tires,
                                                      SDLRPCParameterNameTurnSignal:SDLTurnSignalOff,
                                                      SDLRPCParameterNameVIN:vin,
-                                                     SDLRPCParameterNameWiperStatus:SDLWiperStatusAutomaticHigh},
-                                             SDLRPCParameterNameOperationName:SDLRPCFunctionNameGetVehicleData}} mutableCopy];
+                                                     SDLRPCParameterNameWiperStatus:SDLWiperStatusAutomaticHigh,
+                                                     @"stabilityControlsStatus":stabilityControlsStatus
```

Comment:
`SDLRPCParameterNameStabilityControlsStatus`

Reply:
If I replace the strings with constants declared in another file how can I be sure the constants are correct?
In such case the test will test nothing will it not?